spec/helpers/announcements_helper_spec.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+# Specs in this file have access to a helper object that includes
+# the AnnouncementsHelper. For example:
+#
+# describe AnnouncementsHelper do
+#   describe "string concat" do
+#     it "concats two strings with spaces" do
+#       expect(helper.concat_strings("this","that")).to eq("this that")
+#     end
+#   end
+# end
+RSpec.describe AnnouncementsHelper, type: :helper do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
Delete this file if you don't need anything to test.

(I think you need to test the `strftime` though right)